esmvaltool/cmorizers/obs/cmor_config/ERA-Interim-Land.yml

```diff
@@ -0,0 +1,19 @@
+---
+# Global attributes of NetCDF file
+attributes:
+  dataset_id: ERA-Interim-Land
+  project_id: OBS6
+  tier: 3
+  version: '1'
+  modeling_realm: reanaly
+  source: 'https://apps.ecmwf.int/datasets/data/interim-land/type=an/'
+  reference: 'era-interim-land'
+  comment: |
+    'Contains modified Copernicus Climate Change Service Information {year}'
+
+# Variables to CMORize
+variables:
+  sm:
+    mip: E6hr
```

Comment:
The download script doesnloads monthly means, so I think this should be `Lmon`? Or did you mean to download daily values instead?

Reply:
I am using both `monthly` and `daily` data. I am not personally in charge of downloading the data at our institute (the data is there already and since it can be big, I don't want to duplicate it). So I often run the cmorization by creating symbolic links and adapting the file pattern in the `cmor_config` file. I think it therefore makes most sense if @mattiarighi decides on the `standard` set of data to be downloaded in terms of temporal resolution. For now we can go with `Lmon` as you suggest.

Reply:
We should cmorize the only data actually used (or to be used soon) in the ESMValTool recipes.
So if you need also the daily version of some variable, please add it.

Reply:
So how to accomplish this? For `sm` I need both `daily` and `monthly`. It is not clear to me how this is possible, when in the dictionary one specifies one `mip` per variable. So I can not simply do: 
```
variables:
  sm:
    mip: E6hr
  sm:
    mip: Lmon
```

Reply:
@mattiarighi / @bouweandela : I outline here what I plan to do, please comment if you have a better idea. I found out that ERA-Interim-Land is only available in daily (or actually 6 hourly) resolution. Therefore, I need to derive the monthly resolution myself, and I think I will specify this here. Of course in order for  this to work I also have to modify the ERA-Interim cmorization script.

Reply:
> So how to accomplish this? For `sm` I need both `daily` and `monthly`. It is not clear to me how one can achieve this when in the dictionary one specifies one `mip` per variable. So I can not simply do:

I now implemented support for monthly means for ERA-Interim-Land, this question remains open. Does it deserve its own issue, or is it closely related to any of the currently open issues?

Reply:
You can do:

```yaml
variables:
  sm_daily:
    short_name: sm
    mip: E6hr
  sm_monthly:
    short_name: sm
    mip: Lmon
```
to use two variables with the same short_name in the variables section.

Note that there is a [`monthly_statistics`](https://esmvaltool.readthedocs.io/projects/esmvalcore/en/latest/esmvalcore/preprocessor.html#monthly-statistics) preprocessor function which you could use to convert the 6hourly data to monthly means. Maybe it would be nicest to create a recipe that does this conversion and put it in esmvaltool/recipes/cmorizers? I also started working on something similar for ERA5 in https://github.com/ESMValGroup/ESMValTool/pull/1432.

Reply:
Thanks @bouweandela. I'm not there yet, in my case the coordinates in the `definition` are labelled `longitude`, `latitude`, `time` and not `X`, `Y`, `T`, so this goes wrong in the function `_fix_coordinates`.  I do not understand why this is different between the two datasets.

Reply:
I just pushed a version that works for ERA-Interim-Land. This is not how it should be implemented (since it will break ERA-Interim), but it hopefully shows the problem. Can you have a look at it and point me in a direction how to solve this discrepancy between ERA-Interim-Land (variable `sm`) and ERA-Interim? Or maybe @SarahAlidoost has an idea?

Reply:
> Thanks @bouweandela. I'm not there yet, in my case the coordinates in the definition are labelled longitude, latitude, time and not X, Y, T, so this goes wrong in the function _fix_coordinates. I do not understand why this is different between the two datasets.

The way the CMOR tables were read for CMIP6 type tables and CMIP5 type tables is inconsistent, I've made a pull request to fix that here https://github.com/ESMValGroup/ESMValCore/pull/413. The reason that you ran into the problem for this particular variable is that it is a custom variable, and those are CMIP5 style tables.

Reply:
Thanks. Works fine now in combination with https://github.com/ESMValGroup/ESMValCore/pull/413